cmd/juju/model/configcommand_test.go

```diff
@@ -34,15 +34,7 @@ func (s *ConfigCommandSuite) TestInit(c *gc.C) {
 		errorMatch string
 		nilErr     bool
 	}{
-		{ // Test set
```

Comment:
Yes because as per PR description, we now use the same --config processing as for bootstrap where an attribute may be specified more than once and last one wins. This allows a file to be used and then you can override just one or two values from that file with extra --config etc.